tutor/commands/jobs.py

```diff
@@ -40,6 +41,15 @@ def _add_core_init_tasks() -> None:
             ("mysql", env.read_core_template_file("jobs", "init", "mysql.sh"))
         )
     with hooks.Contexts.APP("lms").enter():
+        hooks.Filters.CLI_DO_INIT_TASKS.add_item(
```

Comment:
now let's hope that no one will want to bind-mount edx-platform in the cms only :crossed_fingers: it's really a edge case, and I don't want to add another init task to handle it. If I'm wrong we'll come back to this comment.

Reply:
I think we should be OK here because `launch` cannot be limited to just cms. They would only really run into this edge case if they tried to skip launch and instead just run:

```
tutor dev do --mount=edx-platform init --limit=cms
```

🤞🏻